lib/matplotlib/backends/__init__.py

```diff
@@ -14,22 +14,21 @@
            'new_figure_manager', 'backend_version']
 
 backend = matplotlib.get_backend() # validates, to match all_backends
+if backend.startswith('module://'):
+    backend_name = backend[9:]
+else:
+    backend_name = 'matplotlib.backends.backend_' + backend.lower()
 
-def pylab_setup():
-    'return new_figure_manager, draw_if_interactive and show for pylab'
+def get_backend():
```

Comment:
I don't fully understand the difference between ``matplotlib.use`` and ``matplotlib.pyplot.swtich_backend``, I think for now we could make ``matplotlib.backends.get_backend`` a private method and leave the rest of the discussion for a later PR?